.travis.yml

```diff
@@ -3,7 +3,7 @@ language: php
 os: linux
 
 # Using xenial which will use MySQL 5.7 by default
```

Comment:
Perhaps the comment should be updated or removed?

Reply:
@emteknetnz Maybe blast this comment.